fees/steamm/index.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Adapter,
+  FetchOptions,
+} from "../../adapters/types";
+import { CHAIN } from "../../helpers/chains";
+import fetchURL from "../../utils/fetchURL";
+
+const suilendFeesURL = 'https://api.suilend.fi/steamm/daily/fees';
+
+interface DailyStats {  
+  protocolFeesUsd: string;
+  poolFeesUsd: string;
+}
+
+const methodology = {
+  Fees: 'Total fees paid from swaps',
+  ProtocolReveneue: 'The portion of the total fees going to the STEAMM treasury'
+}
+
+const fetchSteammStats = async ({ endTimestamp }: FetchOptions) => {
+  const url = `${suilendFeesURL}?ts=${endTimestamp}`
+  const stats: DailyStats = (await fetchURL(url));
+  const dailyRevenue = parseFloat(stats.protocolFeesUsd) + parseFloat(stats.poolFeesUsd)
```

Comment:
why are you counting pool fees that go to LP as revenue here? only the protocol fee should be counted as revenue https://docs.llama.fi/list-your-project/other-dashboards/dimensions